src/main/java/org/skriptlang/skript/lang/structure/Structure.java

```diff
@@ -99,8 +99,6 @@ public int compareTo(@NotNull Structure.Priority o) {
 	@Deprecated
 	@ApiStatus.ScheduledForRemoval
 	public final EntryContainer getEntryContainer() {
-		if (entryContainer == null)
-			throw new IllegalStateException("This Structure hasn't been initialized!");
 		return entryContainer;
```

Comment:
```suggestion
		if (entryContainer == null)
			throw new IllegalStateException("This Structure hasn't been initialized!");
		return entryContainer;
```
Reverting this change. May have been accidental during conflict resolution.